README.md

```diff
@@ -25,6 +25,12 @@ Once you have Ruby and all the other dependencies installed, you can run the fol
 gem install bundler jekyll
 ```
 
+You will also need to install WeBrick Gem if you are using Ruby version 3.0.0 or higher to start the local server.
+
+```ruby
+gem install webrick
```

Comment:
Instead of adding an extra command let's just add this to the `gem install bundler jekyll` line above.

Reply:
Yeah sure sir and I have made a new pull request